src/commands/server/news/edit.ts

```diff
@@ -141,15 +125,14 @@ export default class Edit {
         .setFooter({ text: `Updated news from ${guild.name}` })
         .setColor(genColor(200));
 
-      const subscribedNewsChannel = await newsTable?.get(`${guild.id}.channel`).then(
-        (channel) => channel as { channelId: string; roleId: string } | null
-      ).catch(() => {
-        return {
-          channelId: null as string | null,
-          roleId: null as string | null
-        };
-      });
-      if (Boolean(subscribedNewsChannel.channelId)) {
+      const subscribedNewsChannel = await newsTable
+        ?.get(`${guild.id}.channel`)
```

Comment:
```suggestion
      const subscribedNewsChannel = await newsTable?
        .get(`${guild.id}.channel`)
```
That character belongs there I think ;)